dev-util/vulkan-headers/vulkan-headers-1.2.133.ebuild

```diff
@@ -3,29 +3,26 @@
 
 EAPI=7
 
-inherit cmake-utils
+MY_PN=Vulkan-Headers
+inherit cmake
 
-if [[ "${PV}" == "9999" ]]; then
-	EGIT_REPO_URI="https://github.com/KhronosGroup/Vulkan-Headers.git"
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/KhronosGroup/${MY_PN}.git"
 	inherit git-r3
 else
-	if [[ -z ${SNAPSHOT_COMMIT} ]]; then
-		MY_PV=v${PV}
-		MY_P=Vulkan-Headers-${PV}
-	else
-		MY_PV=${SNAPSHOT_COMMIT}
-		MY_P=Vulkan-Headers-${SNAPSHOT_COMMIT}
-	fi
+	SRC_URI="https://github.com/KhronosGroup/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~ppc64 ~x86"
-	SRC_URI="https://github.com/KhronosGroup/Vulkan-Headers/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
-	S="${WORKDIR}"/${MY_P}
+	S="${WORKDIR}"/${MY_PN}-${PV}
 fi
 
-RDEPEND="!<dev-util/vulkan-tools-${PV}
-	 !<media-libs/vulkan-layers-${PV}"
-
 DESCRIPTION="Vulkan Header files and API registry"
 HOMEPAGE="https://github.com/KhronosGroup/Vulkan-Headers"
 
 LICENSE="Apache-2.0"
 SLOT="0"
+
+BDEPEND=">=dev-util/cmake-3.10.2"
+RDEPEND="
+	!<dev-util/vulkan-tools-1.1.124
+	!<media-libs/vulkan-layers-1.1.125
```

Comment:
Are the numbers above correct? I thought the point was to avoid anything older than the current version for the headers?

Reply:
The blockers look entirely wrong to me. Removing them...